api/read/get_cert_status_changes/index.md

```diff
@@ -31,10 +31,10 @@ This method optionally supports paging, to iterate over all results,
   <result ID="1">
     <TIME>13-Sep-2022 09.21.26</TIME>
     <UNIX_TIME>1663060886</UNIX_TIME>
-    <SITE>GRIDOPS-GOCDB</SITE>
+    <SITE>Izola MRF</SITE>
     <OLD_STATUS>Uncertified</OLD_STATUS>
     <NEW_STATUS>Certified</NEW_STATUS>
-    <CHANGED_BY>/C=UK/O=eScience/OU=CLRC/L=RAL/CN=example user</CHANGED_BY>
+    <CHANGED_BY>/C=UK/O=eScience/OU=CLRC/L=RAL/CN=Rowan Moss</CHANGED_BY>
```

Comment:
Can you change this to one of the fake users?

Reply:
ok